### ISSUE TITLE
FutureWarning re: sklearn.semi_supervised.label_propagation

### ISSUE THREAD

**Issue by bbookman** [EFord36/normalise#119]
` FutureWarning: The sklearn.semi_supervised.label_propagation module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.semi_supervised. Anything that cannot be imported from sklearn.semi_supervised is now part of the private API.`

**Comment by dimaelzein** [comment]
Hello, I uninstalled my current scikit-learn verion and installed the v0.23.1 
Yet, I sill have the following error "No module named 'sklearn.semi_supervised.label_propagation' "
Any idea what might be the problem?

**Comment by ntdat017** [comment]
I found that new update the v0.23.1 will raise the error. You need downgrade scikit-learn versiont to <= v0.22.1 to fix it.
